client/src/modules/list/components/Items/model/reducer.js

```diff
@@ -73,23 +74,44 @@ const items = (state = {}, action) => {
     }
     case ItemActionTypes.UPDATE_SUCCESS: {
       const {
-        payload: { description, isOrdered, name, editedBy, _id: itemId }
+        payload: {
+          _id: itemId,
+          description,
+          editedBy,
+          isArchived,
+          isOrdered,
+          name,
+          votesCount
+        }
       } = action;
-
       const previousItem = state[itemId];
+      const previousName = previousItem.name;
       const previousDescription = previousItem.description;
+      const previousIsArchived = previousItem.isArchived;
+      const previousVotesCount = previousItem.votesCount;
+      const previousIsOrdered = previousItem.isOrdered;
```

Comment:
do you need all these `previous` consts? they are used only once and it is clear when you see just `previousItem.SOMETHING` that it is previous value